lms/djangoapps/teams/views.py

```diff
@@ -1365,11 +1365,14 @@ class MembershipBulkManagementView(GenericAPIView):
     """
     View for uploading and downloading team membership CSVs.
     """
+
+    authentication_classes = (OAuth2AuthenticationDeprecated, SessionAuthentication)
+    permission_classes = (permissions.IsAuthenticated, IsCourseStaffInstructor)
+
     def get(self, request, **_kwargs):
         """
         Download CSV with team membership data for given course run.
         """
-        self.check_access()
```

Comment:
I think check_access was only used here. If it's no longer used, we can remove it.

Reply:
Turns out i did need it! thanks for the comment and making me take another look :cold_sweat: